projects/wikimedia.org.yaml

```diff
@@ -8,33 +8,44 @@ paths:
             title: Wikimedia REST API
             description: >
                 This API provides cacheable and straightforward access to
-                Wikimedia content and data, in machine-readable formats. Each
-                entry point has explicit stability markers to inform you about
-                development status and change policy, according to [our API
-                version policy](https://www.mediawiki.org/wiki/API_versioning).
+                Wikimedia content and data, in machine-readable formats.
```

Comment:
Some of the data is cross-project as well. For licensing purposes, it shouldn't matter anyway, as differing licenses are called out explicitly.